BLOGPOST.md

```diff
@@ -0,0 +1,38 @@
+# Introducing the SAP Cloud SDK for AI (JavaScript/TypeScript) 🎉
+
+We are excited to announce the initial release of the [SAP Cloud SDK for AI](https://github.com/SAP/ai-sdk-js#readme)! It simplifies the integration of generative AI capabilities into your SAP Business Technology Platform (BTP) applications.
+
+The SDK allows you to leverage the SAP AI Core and SAP AI Launchpad, enabling capabilities like templating, grounding, data masking, and more. In this blog post, we’ll introduce you to the key packages and their features.
+
+## Key Features
+### AI Orchestration with `@sap-ai-sdk/orchestration`
+The [@sap-ai-sdk/orchestration](https://github.com/SAP/ai-sdk-js/tree/main/packages/orchestration#readme) package enables generative AI orchestration, allowing you to configure templating, content filtering, and data masking for your applications. With its orchestration service, you can streamline workflows and manage AI models within SAP AI Core.
```

Comment:
```suggestion
The [@sap-ai-sdk/orchestration](https://github.com/SAP/ai-sdk-js/tree/main/packages/orchestration#readme) package enables generative AI orchestration, allowing you to configure templating, content filtering, and data masking for your applications. With the orchestration service, you can streamline workflows and manage AI models within SAP AI Core.
```
[question] It's unclear what kind of workflows are we talking about.
Also, we don't really manage models with the orchestration service right?

Reply:
we configure:
- model name
- model options
- model versions